### PR TITLE
fix: always create `privateBytes` and `sharedBytes`fields in the memory info structures

### LINKED ISSUE #13385: Enable disabled "app module getAppMetrics() API" test
It has been disabled after the Chromium 66 upgrade.
The test fails on Windows with `"private bytes is not > 0"` error message (`privateBytes` is `undefined`).

The method returns info in [MemoryInfo structure](https://electronjs.org/docs/api/structures/memory-info) format.
The API uses [`ProcessMetrics::GetMemoryBytes`](https://chromium.googlesource.com/chromium/src/+/66.0.3359.181/base/process/process_metrics_win.cc#84) under the hood to retrieve the information from the OS.

### PR DESCRIPTION
**Pull request by torycl**
Test seems to pass now even with enabled DCHECK in process_metrics_win.cc in Chromium.

Fixes #13385.

### CODE REVIEW

**Comment by torycl** [comment]
Related issue:
https://github.com/electron/electron/issues/13414

**Comment by alexeykuzmin** [comment]
@torycl The test still fails cause it expects the values to be always strictly greater than `0`.

**Comment by alexeykuzmin** [comment]
[`process.getProcessMemoryInfo()`](https://github.com/electron/electron/blob/master/docs/api/process.md#processgetprocessmemoryinfo) docs should be changed too.

File: docs/api/structures/memory-info.md [hunk 1]
```diff
@@ -5,8 +5,9 @@
 * `peakWorkingSetSize` Integer - The maximum amount of memory that has ever been pinned
   to actual physical RAM. On macOS its value will always be 0.
 * `privateBytes` Integer - The amount of memory not shared by other processes, such as
-  JS heap or HTML content.
+  JS heap or HTML content. Value 0 means the value cannot be obtained in the moment.
```
Comment (alexeykuzmin) [inline docs/api/structures/memory-info.md]: "at the moment"

**Comment by alexeykuzmin** [comment]
@codebytere 
Check this out https://windows-ci.electronjs.org/project/AppVeyor/electron/build/1.0.6223/job/wuu1lqaoigqrnyvk#L1824

> not ok 46 app module getAppMetrics() API returns memory and cpu stats of all running electron processes
Error: Could not call remote function 'anonymous'. Check that the function signature is correct. Underlying error: The 'privateBytes' property has been deprecated.

The test fails because of a deprecation warning you've added recently.
Can you please take a look and fix it?

**Comment by codebytere** [comment]
What do we plan to do with it in the future? if we are still planning to deprecate it then our path forward is different than if we're transitioning away from that...we know why the test is failing so either we should un-deprecate or not test that for the time being i would say

**Comment by alexeykuzmin** [comment]
@codebytere 
In the 3.0.0 release those fields should be deprecated but still be available. It means that we still have to have tests for them.
So it looks like we shouldn't use `process.throwDeprecation = true` in the tests.

**Comment by alexeykuzmin** [comment]
@torycl Can you please rebase the PR onto the latest master?

**Comment by ckerr** [comment]
@torycl bump

**Comment by codebytere** [comment]
Closing as abandoned.

**Pull request closed without merging**